<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigEditor</class>
 <widget class="QDockWidget" name="ConfigEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabs">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabScreen">
       <attribute name="title">
        <string>Screen</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Monitor:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="4">
         <widget class="QComboBox" name="screens"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>X:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="x">
          <property name="minimum">
           <number>-9999</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Width:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QSpinBox" name="w">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Y:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="y">
          <property name="minimum">
           <number>-9999</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Height:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QSpinBox" name="h">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0" colspan="5">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:0.8em; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Using the mouse:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;- &lt;span style=&quot; font-weight:600;&quot;&gt;wheel&lt;/span&gt; : resize horizontally ;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;-&lt;span style=&quot; font-weight:600;&quot;&gt; Ctrl + wheel&lt;/span&gt; : resize vertically ;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;- &lt;span style=&quot; font-weight:600;&quot;&gt;first&lt;/span&gt; &lt;span style=&quot; font-weight:600;&quot;&gt;double-clic : &lt;/span&gt;expand horizontally ;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;- &lt;span style=&quot; font-weight:600;&quot;&gt;second&lt;/span&gt; &lt;span style=&quot; font-weight:600;&quot;&gt;double-clic&lt;/span&gt; :  expand vertically to the top ;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Rotation:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>degres</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="rotation">
          <property name="minimum">
           <double>-360.000000000000000</double>
          </property>
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabStyles">
       <attribute name="title">
        <string>Styles</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::RestoreDefaults|QDialogButtonBox::Save</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Saved presets:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QComboBox" name="presets">
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>x</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ConfigEditor</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>y</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ConfigEditor</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>h</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ConfigEditor</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>screens</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ConfigEditor</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>ConfigEditor</receiver>
   <slot>onClicked(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>w</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ConfigEditor</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>presets</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ConfigEditor</receiver>
   <slot>presetChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotation</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ConfigEditor</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>apply()</slot>
  <slot>onClicked(QAbstractButton*)</slot>
  <slot>presetChanged(int)</slot>
 </slots>
</ui>
